hashintel/hash
# Support property arrays in entity editor

## 🌟 What is the purpose of this PR?

This PR adds an important missing functionality to entity editor. Which is supporting properties with `allow multiple values` checked. Entity editor was supporting arrays before, but that was only for `references to array custom property array definitions`, not `allow property values`


## 🔗 Related links

- [Asana task](https://app.asana.com/0/1203363157432084/1203549170064068/f) _(internal)_

## ❓ How to test this?


1.  Checkout the branch / view the deployment
1.  Open entity editor
2. Create a type with some properties that ticks `allow multiple values` field
3. Create an entity from that type
4. Confirm that entity editor works as expected with `allow multiple values`

## 📹 Demo

Check the list icons on the first column of the entity editor.

<img width="1209" alt="Screenshot 2022-12-13 at 22 29 32" src="https://user-images.githubusercontent.com/39553853/207428071-cc2ed7ae-caeb-47ed-bde4-fc7c5c785f31.png">

<img width="1200" alt="Screenshot 2022-12-13 at 22 29 37" src="https://user-images.githubusercontent.com/39553853/207428079-546bf0fb-4b8d-4157-a02b-8e94e9d0ee03.png">
